apollo-client/v3/schema-driven-testing/src/client.ts

```diff
@@ -0,0 +1,17 @@
+import { ApolloClient, HttpLink, InMemoryCache } from "@apollo/client";
+
+const httpLink = new HttpLink({
+  uri: "https://main--hack-the-e-commerce.apollographos.net/graphql",
+});
+
+export const client = new ApolloClient({
+  cache: new InMemoryCache(),
+  link: httpLink,
+});
+
+export const makeClient = () => {
+  return new ApolloClient({
+    cache: new InMemoryCache(),
+    link: httpLink,
+  });
+};
```

Comment:
```suggestion
export const makeClient = () => {
  return new ApolloClient({
    cache: new InMemoryCache(),
    link: httpLink,
  });
};

export const client = makeClient();
```

[nit] Since I figure this should model what we'd ask of our users, let's consider using the `makeClient` function to create the singleton instance. This will be especially nicer for users that have large link chains or custom handling.

Reply:
Good call! Updated 👍